hnn_core/cell_response.py

```diff
@@ -138,75 +142,9 @@ def __eq__(self, other):
                        for trial in other._spike_times]
         return (times_self == times_other and
                 self._spike_gids == other._spike_gids and
-                self._spike_types == other._spike_types)
-
-    def __getitem__(self, gid_item):
```

Comment:
do we need to document this in `whats_new.rst` as an API change?

Reply:
Pinging @ntolley ... I see the PR is set to MRG but it's waiting for response to this comment?

Reply:
I'm falling behind :smile: 

The main reason for getting rid of this functionality is that I think it adds a lot of complexity and maintenance difficulties for not much payoff. This was added in the early days of my development, so it's really not documented